index.html

```diff
@@ -0,0 +1,229 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Mobile Version</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <section class="mobile">
+      <div class="head">
+        <h1 class="title">My Recent Works</h1>
+        <hr />
+      </div>
+      >
+
+      <div class="image"></div>
+
+      <section class="banner">
+        <h2>Multi-Post Stories</h2>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+        <ul class="bn-buttons bn-list">
+          <li><button class="blue" type="button">css</button></li>
+          <li><button class="orange" type="button">html</button></li>
+          <li><button class="pink" type="button">bootstrap</button></li>
+          <li><button class="red" type="button">Ruby</button></li>
+        </ul>
+        <div class="project see-pr">
+          <button class="pr-button" type="button">See Project</button>
+        </div>
+      </section>
+
+      <section class="works">
+        <div class="work1">
+          <h1 class="hash">
+            Profesional Art <br />
+            Printing Data
+          </h1>
+          <p class="pi">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+          <ul class="buttons ul">
+            <li class="html"><button type="button">html</button></li>
+            <li class="boot"><button type="button">bootstrap</button></li>
+            <li class="ruby"><button type="button">Ruby</button></li>
+          </ul>
+          <div class="project list">
+            <button class="pr-button" type="button">See Project</button>
+          </div>
+        </div>
```

Comment:
- [ ] [OPTIONAL]: To keep your HTML semantic, you should consider changing this div to an `<article>`, you can read more about it [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article).
- [ ] [OPTIONAL]: Having more than one `<h1>` is allowed by the HTML specification, but is not considered a best practice, you should consider changing those to an `<h2>` or `<h3>`, you can read more about it [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements).

Reply:
Done.